qstash/api/messages/get.mdx

```diff
@@ -17,11 +17,45 @@ authMethod: "bearer"
 
 <RequestExample>
 
-```sh
+```sh curl
 curl https://qstash.upstash.io/v2/messages/msg_123 \
   -H "Authorization: Bearer <token>"
 ```
 
+```js node
+const response = await fetch("https://qstash.upstash.io/v2/messages/msg_123", {
+  headers: {
+    Authorization: "Bearer <token>",
+  },
+});
+```
+
+```go Go
```

Comment:
Can you make the order as curl, Node, Python, Go here to be consistent with the rest?